sessionctx/variable/tidb_vars.go

```diff
@@ -131,6 +131,9 @@ const (
 
 	// TiDBCheckMb4ValueInUtf8 is used to control whether to enable the check wrong utf8 value.
 	TiDBCheckMb4ValueInUtf8 = "tidb_check_mb4_value_in_utf8"
+
+	// TiDBCheckMb4ValueInUtf8 is used to control whether to enable the check wrong utf8 value.
+	TiDBTreadOldVersionUTF8AsUTF8MB4 = "tidb_treat_old_version_utf8_as_utf8mb4"
```

Comment:
It's weird that the 'UTF8' in this name is all upper case but the variable name 'TiDBCheckMb4ValueInUtf8' uses pascal style('Utf8'), better to make them unified.

Reply:
done.